src/components/TimeRelated.tsx

```diff
@@ -11,10 +11,9 @@ export interface IProps {
 }
 
 const SetHours = (timeData: string) => {
-    const currentDate = new Date();
-    const inputDate = new Date(timeData);
-
-    return isSameDay(inputDate, currentDate) ? 'Сейчас' : format(new Date(timeData), 'HH:mm');
+    const currentHour = getHours(new Date());
+    const inputHours = getHours(new Date(timeData));
+    return inputHours === currentHour ? 'Сейчас' : format(new Date(timeData), 'HH:mm');
```

Comment:
It is better not to create several instances of the same object, in this case - objects of Date.